py/mockbuild/buildroot.py

```diff
@@ -175,17 +175,17 @@ def doChroot(self, command, shell=False, nosync=False, *args, **kargs):
         env = dict(self.env)
         if nosync and self.nosync_path:
             env['LD_PRELOAD'] = self.nosync_path
-        if util.USE_NSPAWN:
+        if util.USE_NSPAWN or kargs.get('uid', 1000) == 0:
```

Comment:
It's just a placeholder. I could probably use `-1` here as well.
If we elevated to root `kargs['uid'] == 0` otherwise kargs may not even have 'uid' key (yes, it does happen).
But if `-1` is better, I can change it.